packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -79,6 +83,8 @@ module.exports = class SingleSpaReactGenerator extends PnpmGenerator {
       delete packageJson.devDependencies["webpack-config-single-spa"];
       // Will be replaced by webpack-config-single-spa-react-ts
       delete packageJson.devDependencies["webpack-config-single-spa-ts"];
+
+      packageJson.types = `dist/${this.options.orgName}-${this.options.projectName}.d.ts`;
```

Comment:
In case people publish the packages to npm, the `"types"` field should be properly set